### DISCUSSION TITLE
I cannot see the full error message when the error is long

### DISCUSSION THREAD

**Discussion by vesper8** [spatie/ray#408] [General]
I noticed that for many exceptions that show up on RAY I cannot see the full message error. This is very annoying and the only way I can see the full message is to 'share with ignition' and then click again to the copy the url, open the ignition link and then click one more time to expand the error. That's like 5 clicks and a website I have to open just to see the full error.

This message is also not shown in the stack trace, it's only on the very first line that it's visible

I would like you to ask to please make the long error message expand in RAY when you click on it.. and to optionally also be able to disable those sharing buttons

Many thanks

**Comment by jreviews** [comment]
I also think this would be a nice improvement. I end up having to copy the exception message and pasting it elsewhere to be able to read it.

![image](https://user-images.githubusercontent.com/31069418/115304170-9d094800-a132-11eb-8664-f1fa16b6224d.png)

**Reply by freekmurze** [reply]
@AdrianMrn maybe we can show the full error in an small overlay on hover and toggle the full error on click?

**Comment by AdrianMrn** [comment]
Clicking the message will now expand the exception message, hovering over it will also show it in full!

**Reply by jreviews** [reply]
Thanks!
